ui-react/src/components/accounts/EditImporter.tsx

```diff
@@ -15,14 +15,23 @@ export default function EditImporter({
   handleDelete,
   importer
 }: EditImporterProps) {
-    const [loginFields, setLoginFields] = useState<Record<string, string>>(importer.loginFields || {});
+    const [loginFields, setLoginFields] = useState<Record<string, string>>(importer.loginFields || {username: '', password: '', userCode: '', id: '', num: '', card6Digits: ''});
     const [active, setActive] = useState<boolean>(importer.active);
-    const onSaveClicked = async () => {
-        await handleSave({
-            ...importer,
-            active,
-            loginFields
-        });
+    const [validated, setValidated] = useState(false);
+    const onSaveClicked = async (event) => {
+        const form = event.currentTarget
+        if (form.checkValidity() === false) {
+            event.preventDefault();
+            event.stopPropagation();
+        } else {
+            await handleSave({
+                ...importer,
+                active,
+                loginFields
+            });
+        }        
+        
+        setValidated(true);
```

Comment:
In my optionion, using a Form is unnecessary in React.

Instead, you can just validate the Form on `onSaveClicked` and change the `validated` state.
Also you can enable/disable the Save button by the `validated` state.

This will hekp you avoiding taking care of `preventDefault` and `stopPropagation`.